servers/rendering/renderer_rd/shader_compiler_rd.cpp

```diff
@@ -1373,6 +1373,9 @@ ShaderCompilerRD::ShaderCompilerRD() {
 	//builtins
 
 	actions[RS::SHADER_SPATIAL].renames["TIME"] = "time";
+	actions[RS::SHADER_SPATIAL].renames["PI"] = "3.1415926535";
```

Comment:
This is all dead code it is probably just kept as reference and will soon be deleted. You shouldn't add anything here.